### ISSUE TITLE
Clear the Screen should do Exactly That

### ISSUE THREAD

**Issue by VictorXLR** [Microsoft/vscode#30638]
- VSCode Version: Latest
- OS Version: Windows 10 

Steps to Reproduce:

1. Open VSCODE terminal `CMD`
2. Run a few Commands,
3. Type `CLS`

it should clear the screen and not put everything on a new line. 

Might not be a serious issue but it bugs the hell out of me  :)

Reproduces without extensions: Yes

**Comment by Tyriar** [comment]
This is being discussed upstream https://github.com/sourcelair/xterm.js/issues/106

**Comment by ghiscoding** [comment]
Not sure if this is exactly the same behavior as what the issue was opened with but here's my observations. 

Doing a `cls` does clear the screen but it's annoying to see the previous history coming back after running another command. For example, if I run a Unit Test and do a `cls`, I don't expect to see previous tests coming back after a `cls` but they do and this makes the terminal a little hard to follow. I'm running version `1.18.0-insiders` which has the new terminal rendering engine.

**Comment by Spown** [comment]
eyep, have to destroy terminal before every test call. upstream or not this is ridiculous.

**Comment by suadev** [comment]
same here!  

I have to create a new instance of terminal after every 'dotnet test' command!

**Comment by TheKojukinator** [comment]
Just ran in to this issue myself. Every time I F5 my PowerShell script, all the previous output that used to be in the terminal returns, along with the new output. If I perform cls/Clear-Host it goes away until I F5 again, and the old content returns with new output appended. And so on...

Here's my about window:
![image](https://user-images.githubusercontent.com/30471780/39490094-a4e92722-4d55-11e8-9166-cea9fe223ede.png)

**Comment by juancrl** [comment]
Any update ?
The console cannot provide wrong information !

**Comment by TheKojukinator** [comment]
https://github.com/xtermjs/xterm.js/issues/106 got closed on June 1st, as the problem appears to be with https://github.com/rprichard/winpty.
https://github.com/Microsoft/vscode/issues/45693 lists all the winpty issues affecting vscode, and this one is among them.

I checked the winpty repo for open issues mentioning "cls" or "clear" but nothing stands out as related to this. I don't have enough expertise to open an issue with winpty and provide details that would be relevant, but maybe that's what needs to be done to move forward on this?
